core/strings/ascii_set.odin

```diff
@@ -2,10 +2,20 @@
 package strings
 
 import "core:unicode/utf8"
-
+// Ascii_Set is designed to store ASCII characters efficiently as a bit-array
```

Comment:
Can't multi select lines on mobile unfortunately.
Since tabbing is significant (it determines code blocks) you should do all the documentation within a block comment, i.e. /* */
Look at package dynlib for example.